src/shady-css/lexer.js

```diff
@@ -0,0 +1,141 @@
+import { matcher, tokenType } from './common';
+
+class Lexer {
```

Comment:
totally incredibly minor naming nit: I like the name Tokenizer for this class since it creates a list of tokens and all the methods are tokenize*. I realize they are synonyms though